packages/components/src/mobile/link-picker/styles.native.scss

```diff
@@ -16,18 +16,6 @@
 	justify-content: center;
 }
 
-.clearIcon {
-	margin-left: 16;
```

Comment:
Not sure if it's needed but I'm wondering if we should carry this style to the `cellTextInputContainer`, wdyt?

Reply:
As you can see in the following screenshot, the close button looks too close to the text input which could lead users to tap it when trying to interact with the text caret, so we might consider giving extra padding.

<img src=https://user-images.githubusercontent.com/14905380/146238410-6c101f2a-8026-4fac-bff2-4f87136c84cb.png width=400>

Reply:
I think this is a good idea. I've updated the styles to match.

Reply:
When doing some testing I cleared the input a couple of times by mistake when trying to select text, I noticed that the bounds of the clear button might interfere with the text input itself. This is not an issue when the text is aligned to the left but can be a major problem when is aligned to right. I'm wondering if we could tweak the button to prevent this behavior, wdyt?

<img width="495" alt="Screenshot 2021-12-16 at 20 10 59" src="https://user-images.githubusercontent.com/14905380/146433679-c2c02c8d-ec6f-422b-a4d2-7f733e493528.png">

Reply:
I think this is a good callout, and I've updated the behavior to be more compatible with right aligned text. Initially, I had pulled in the style directly from LinkPicker, which was `margin-left: 16;`, and that felt a bit too wide for right aligned text. Now, after interacting with it a bit, it feels like a good amount, and provides enough buffer space for the user to cleanly interact with both the text and the button, without interference. Thanks for the suggestion, much appreciated. 👍

Reply:
Awesome, I'll perform another test on this part as a double-check before solving the issue, thanks for addressing it 🙇 .

Reply:
I've just tested it and I can confirm that the clear button no longer produces this issue 🎊 .